crates/engine/src/ext.rs

```diff
@@ -257,6 +257,17 @@ impl Engine {
         }
     }
 
+    /// Returns the decoded contract storage at the key if any.
```

Comment:
Looks like the comment copied/pasted from `get_storage`

Reply:
I had modified it to reflect that it's the size, how do you think it should be instead?

Reply:
The comment says that it returns the "decoded contract storage" but in fact it returns the size.